detectable/src/main/java/com/synopsys/integration/detectable/detectables/sbt/dot/SbtDotGraphNodeParser.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * detectable
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.detectable.detectables.sbt.dot;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.bdio.model.dependency.Dependency;
+import com.synopsys.integration.bdio.model.externalid.ExternalId;
+import com.synopsys.integration.bdio.model.externalid.ExternalIdFactory;
+
+public class SbtDotGraphNodeParser {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+    private final ExternalIdFactory externalIdFactory;
+
+    public SbtDotGraphNodeParser(final ExternalIdFactory externalIdFactory) {
+        this.externalIdFactory = externalIdFactory;
+    }
+
+    public Dependency nodeToDependency(String id) {
+        String[] pieces = id.trim().split(":");
+        String group = unquote(pieces[0]);
+        String name = unquote(pieces[1]);
+        String version = unquote(pieces[2]);
+        ExternalId externalId = externalIdFactory.createMavenExternalId(group, name, version);
+        return new Dependency(name, version, externalId);
+    }
+
+    public String unquote(String item) {
+        if (item.startsWith("\""))
+            item = item.substring(1);
+        if (item.endsWith("\""))
+            item = item.substring(0, item.length() - 1);
+        return item;
+    }
```

Comment:
There's no risk of an item with internal quotes (that look like external quotes) like "a"="b", is there?

Reply:
That's a very good question, I don't think we will see an SBT project with quotes in the wild but that doesn't mean we won't. As a test I added a quote to the organization and it broke the GraphParser (the third party dot file parser we use) so it doesn't seem like we will be able to handle quotes in the pieces at all. Knowing that, I'm inclined to leave it and if we see issues it looks like it will require more significant changes.